iceberg-handler/src/test/java/org/apache/iceberg/mr/hive/TestHiveShell.java

```diff
@@ -154,6 +155,13 @@ public void closeSession() {
     }
   }
 
+  public String executeExplain(String statement) {
+    List<Object[]> objects = executeStatement(statement);
+    return objects.stream()
+        .map(o -> (String) o[0])
+        .collect(Collectors.joining("\n"));
+  }
+
```

Comment:
👍

Reply:
Needed this during way too many debug sessions :)

Reply:
Leave a comment there not to remove it even if it is not used 😄